<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMainWindow</class>
 <widget class="QMainWindow" name="AdminMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RayTracing | Подсистема «Администратор» | Alpha 1.0v</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Виртуальная лаборатория для 3D-моделирования</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_admin_test">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Доступ к администрированию тестирования</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/admin-test-button.png</normaloff>images/admin-test-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>42</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_admin_panel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Доступ к администрированию пользователей</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/admin-panel-button.png</normaloff>images/admin-panel-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>42</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_admin_theory">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Доступ к администрированию теоретических материалов</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/admin-theory-button.png</normaloff>images/admin-theory-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>42</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_admin_lab">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Доступ к лабораторным работам</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/admin-lab-button.png</normaloff>images/admin-lab-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>42</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_help">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Получить справку</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/help-button.png</normaloff>images/help-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Пользователь</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Профиль</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Выход из профиля</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Контекстная подсказка</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Руководство пользователя</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
